patches/minecraft/net/minecraft/block/RedstoneDiodeBlock.java.patch

```diff
@@ -9,12 +9,11 @@
           func_220059_a(p_220069_1_, p_220069_2_, p_220069_3_, tileentity);
           p_220069_2_.func_217377_a(p_220069_3_, false);
  
-@@ -163,6 +163,8 @@
+@@ -163,6 +163,7 @@
     protected void func_176400_h(World p_176400_1_, BlockPos p_176400_2_, BlockState p_176400_3_) {
        Direction direction = p_176400_3_.func_177229_b(field_185512_D);
        BlockPos blockpos = p_176400_2_.func_177972_a(direction.func_176734_d());
-+      if (net.minecraftforge.event.ForgeEventFactory.onNeighborNotify(p_176400_1_, p_176400_2_, p_176400_1_.func_180495_p(p_176400_2_), java.util.EnumSet.of(direction.func_176734_d()), false).isCanceled())
-+         return;
++      if (net.minecraftforge.event.ForgeEventFactory.onNeighborNotify(p_176400_1_, p_176400_2_, p_176400_1_.func_180495_p(p_176400_2_), java.util.EnumSet.of(direction.func_176734_d()), false).isCanceled()) return;
```

Comment:
So, instead of exiting out... we are just skipping the next line of code?
Oh wait no, we're still exiting but I had to scroll my screen to see the remainder of the code.
You made the code more confusing because you wanted to lower a line.